### DISCUSSION TITLE
How to use git-tags with regex managers

### DISCUSSION THREAD

**Discussion by submac** [renovatebot/renovate#15356] [Request Help]
Hi,

My goal is to set up:
1. custom manager using `regex`
2. use `git-tags` datasource to grab latest dependency versions

Assume I have 2 repos. Both are located on BitBucket Server (Enterprise) and are in the same project.
Repo A has a manifest file that I'm reading with `regex`. This file lists the repo name and the version number of terraform module located in repo B (repo name = tf module name).
Repo B holds terraform module. Relevant commits are tagged with semantic versioning i.e 0.1.0

I pretty much figured out custom manager part:
```
"stats": {
         "managers": {"regex": {"fileCount": 1, "depCount": 1}},
         "total": {"fileCount": 1, "depCount": 1}
       }
```
but I don't know how to read tags from a different repository. Could anyone point me to any documentation or examples or give a hint on how to approach it?

Thanks

**Comment by PhilipAbed** [comment]
You mean this? https://docs.renovatebot.com/modules/manager/regex/

This is used to 
Detect dependency name and version using regex.
set up the `datasource` to look up for new versions

now you want your custom regex manager to detect that the Dependency is actually another Repository?
but the regex manager will try to look it up in the `datasource` and it won't find it.

I'm not sure renovate supports what you want, unless you can add your repository as a `datasource`, so renovate can look up for new versions in the repository itself, but i'm not sure how to do it or even if its possible im still new here

i would suggest you treat it as a private package: https://docs.renovatebot.com/getting-started/private-packages/
hope this is helpful
